backend/api/Models/AccessRequestModel.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Pims.Api.Models
+{
+    public class AccessRequestModel : BaseModel
+    {
+        #region Properties
+        public Guid Id { get; }
+        public UserModel User { get; set; }
+        public IEnumerable<AgencyModel> Agencies { get; set; }
```

Comment:
Only allow for a single agency.  While we support multiple, we will only allow them to select one.

Reply:
However depending on how  you use this in the code.  Still need to review below...  It is possible to be assigned a parent Agency and therefore you would have access to child agencies.

Reply:
I'm wondering if we want to limit this in the database or in the controller/business logic itself? I actually had this as one Agency/Role per request but changed it to conform with the structure of the user entity.

Reply:
We want the DB to support multiple.  The client hasn't been able to confirm, but they currently said it is most likely only one to one.

Reply:
@Fosol So are you ok with the AccessRequest table to be left as-is (except for the table name, I'' fix that), and then the actual controllers will be responsible for enforcing the one Agency/Role policy?

Reply:
Yes I think so.